### PR TITLE
Add function to allow for query execution without validation, fixes #25

### LINKED ISSUE #25: Make it possible to bypass validations
Even though I like the query-validations, there are times when I need to do stuff that the schemas are
not allowing. Example at hand right now is working with `thetaSketches`, http://druid.io/docs/latest/development/extensions-core/datasketches-aggregators.html
which is not currently supported by the schema. As a work around I do this
```clojure
(defn execute*
  "Issue a druid query"
  ([client balance-strategy druid-query http-params]
   (execute* client balance-strategy druid-query http-params false))
  ([client balance-strategy druid-query http-params non-strict?]
   (let [nodes (druid/nodes client)]
     (when-not non-strict?
       (v/validate druid-query (:queryType druid-query)))
     (when (empty? nodes)
       (throw (Exception.
               "No druid node available for query")))
     (http/post (balance-strategy nodes)
                (merge {:body (json/encode druid-query)
                        :as :text
                        :content-type :json}
                       http-params)))))
```
in my code.

Would you want this in cli-druid? if not, please just close this issue.

### PR DESCRIPTION
**Pull request by slipset**
- Using s/defn to allow for client to decide when to use validation
- Add new function (`execute`) to keep backward compatability
- Fixed error in schema for `timeseries`
- Updated README.md
- Remove use of swiss-arrow

### CODE REVIEW

**Comment by slipset** [comment]
Please regard this as a first draft, I'll be happy to discuss the implementation.

File: src/clj_druid/client.clj [hunk 1]
```diff
@@ -79,14 +79,29 @@
...
+  (let [http-params (apply hash-map params)
+        http-body (-> druid-query
+                      (assoc :queryType query-type)
+                      json/write-str)
         nodes (nodes client)]
     (when (empty? nodes)
       (throw (Exception.
               "No druid node available for query")))
-    (http/post (balance-strategy nodes) options)))
+    (http/post (balance-strategy nodes) (merge {:body http-body
```
Comment (gbuisson) [inline src/clj_druid/client.clj]: perhaps you could put the `merge`on the next line instead

File: src/clj_druid/client.clj [hunk 2]
```diff
@@ -79,14 +79,29 @@
 (defn query
   "Issue a druid query"
   [client balance-strategy query-type druid-query & params]
-  (let [params (apply hash-map params)
-        options (-<> (into druid-query {:queryType query-type})
-                     (v/validate query-type)
-                     (json/write-str <>)
-                     {:body <> :as :text}
-                     (merge params))
+  (v/validate druid-query query-type)
```
Comment (gbuisson) [inline src/clj_druid/client.clj]: I would put the `validate`call after the `let`to improve readability

File: README.md [hunk 3]
```diff
@@ -71,7 +71,21 @@ Issue druid queries supplying
...
+   (let [client (connect {:zk {:host "127.0.0.1:2181"
+                          :discovery-path "/druid/discovery"
+                          :node-type "druid:broker"}})]
+     (execute client random q)
+     (close client))
+;; if you want to validate
+   (let [client (connect {:zk {:host "127.0.0.1:2181"
+                          :discovery-path "/druid/discovery"
+                          :node-type "druid:broker"}})]
+     (s/with-fn-validation (execute client random q)) ;; s is plumatic/schema
```
Comment (gbuisson) [inline README.md:86]: I like the fact that you added a new fn, this will help backward compat

**Review by gbuisson — APPROVED** [review]
besides some nit picks, LGTM! I think this PR also closes #8

**Review by msprunck — APPROVED** [review]
LGTM!

**Pull request merged**